stable/traefik/values.yaml

```diff
@@ -41,6 +42,8 @@ service:
   #   key: value
 gzip:
   enabled: true
+daemonset:
+  enabled: false
```

Comment:
I think I'd prefer that we exposed an option `deploymentModel` with allowable values `deployment` and `daemonset`. I say this because, looking toward the future and a hope of supporting a more sophisticated scaling option (master/slave), I wouldn't want this choice to be binary.